ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_indexing.py

```diff
@@ -428,3 +428,34 @@ def test_jax_diag_indices_from(
         on_device=on_device,
         arr=x[0],
     )
+
+
+@handle_frontend_test(
+    fn_tree="jax.numpy.indices",
+    dimensions=helpers.get_shape(min_num_dims=1),
+    dtype=helpers.get_dtypes(),
```

Comment:
You can pass `"numeric"` in the `heleprs.get_dtypes` method because by default it is `"valid"` an that also produces `bool`